### PR TITLE
fix: remove extraneous dollar sign in `keys` output

### LINKED ISSUE #49: Extraneous dollar sign in "keys" output
There exists an extraneous dollar sign (`$`) in the `keys` output, thus escaping the start delimiter of the the (unnecessary) string interpolation by converting `$${` into a literal `${`. This results in a map of output of the exact string as `${base64decode(google_service_account_key.keys[v].private_key)}`, which is surely not the intended output.

I believe this is an unintentional regression introduced in commit 92cac5bf7908e2ee8a44d4d86b6e31548c1f60f3, when the template_file data source was removed and its rendering logic replaced. This, in turn, ended up in the 4.0.1 and later releases.

### PR DESCRIPTION
**Pull request by morgante**
Fixes #49.

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

File: outputs.tf [hunk 1]
```diff
@@ -68,5 +68,5 @@ output "iam_emails_list" {
 output "keys" {
   description = "Map of service account keys."
   sensitive   = true
-  value       = { for k, v in local.names : k => var.generate_keys ? "$${base64decode(google_service_account_key.keys[v].private_key)}" : "" }
+  value       = { for k, v in local.names : k => var.generate_keys ? "${base64decode(google_service_account_key.keys[v].private_key)}" : "" }
```
Comment (savvykms) [inline outputs.tf:71]: Would `base64decode(google_service_account_key.keys[v].private_key)` outside of a string be safe? I'd imagine the input/output is always a UTF-8 string, but I could be wrong.

**Review by savvykms — APPROVED** [review]

File: outputs.tf (hunk 1 above)
Comment (morgante) [reply outputs.tf]: Good point, updated.

**Comment by morgante** [comment]
@bharathkkb Friendly ping.

**Review by bharathkkb — APPROVED** [review]
Sorry missed this 😅

**Pull request merged**